src/styles/main.scss

```diff
@@ -1,3 +1,70 @@
+@import 'utils/variables';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: 710px;
+
+  &__day {
+    width: $box-size;
+    height: $box-size;
+    border: $border-size solid black;
+    background-color: #eee;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    transition-duration: 0.5s;
```

Comment:
specify what exactly you'd like to change on hover (by default transition-property is `all` and it's not a good practice to use `all`)